### PR TITLE
Add ability to configure [testenv] in tox.ini.

### PR DESCRIPTION
**Pull request by icemac**
This is needed for https://github.com/zopefoundation/zope.publisher/pull/58/commits/a897f45ee32214bc09d2d1bdedb75edb97d67de5

### CODE REVIEW

**Review by mgedmin — APPROVED** [review]
LGTM

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

File: config/README.rst [hunk 1]
```diff
@@ -237,21 +242,35 @@ with-sphinx-doctests
...
+The corresponding section is named: ``[coverage]``.
+
 fail-under
   A minimal value of code coverage below which a test failure is issued.
 
 
 Coverage:run options
 ````````````````````
 
+The corresponding section is named: ``[coverage-run]``.
```
Comment (sallner) [inline config/README.rst:254]: I know, it does not belong to this PR, but I believed, that nested structures were one of the selling point of toml. I would have expected the configuration for `coverage run` under `[coverage.run]`. But that requires of course some change of code.

**Review by sallner — APPROVED** [review]
👍🏾

File: config/README.rst (hunk 1 above)
Comment (icemac) [reply config/README.rst:254]: I thought this would make the configuration file unnecessary complex and I do not really like the syntax (and its variants) used for nested structures.
